README.md

```diff
@@ -0,0 +1,80 @@
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+# Apache Daffodil SBT Plugin
+
+Plugin to enable Daffodil capabilities for DFDL schema projects.
+
+## Enable
+
+To enable the plugin, add the following to `project/plugins.sbt`:
+
+```scala
+addSbtPlugin("org.apache.daffodil" % "sbt-daffodil" % "<version>")
+```
+
+## Features
+
+### Saved Parsers
+
+This plugin adds the ability to create and publish saved parsers of a schema.
+
+For each saved parser to generate, add an entry to the
+`daffodilPackageBinInfos` setting. This setting is a Seq of 3-tuples made up of
+the resource path to the schema, an optional root element to use in that
+schema, and an optional name that is added to the artifact classifier to
+differentiate multiple saved parsers. For example:
+
+```scala
+daffodilPackageBinInfos := Seq(
+  ("/com/example/xsd/mainSchema.dfdl.xsd", None, None)
+  ("/com/example/xsd/otherSchema.dfdl.xsd", Some("otherRoot"), Some("other"))
+)
+```
+
+You must also define which versions of Daffodil to build comptiable saved
+parsers using the `daffodilPackageBinVersions` setting. For example, to build
+saved parsers for Daffodil 3.6.0 and 3.5.0:
+
+```scala
+daffodilPackageBinVersions := Set("3.6.0", "3.5.0")
```

Comment:
Can I compile for a snapshot release (like the one I'm working on) somehow also?

Reply:
Yep, just run `sbt publishLocal` from the daffodil repo and add the snapshot version to `daffodilPackageBinVersions`. The plugin downloads the Daffodil version jars using SBT dependency resolution, so as long as SBT can find the jars it should work.

Reply:
Will it be "370" or "370SNAPSHOT" or something like that in the binary files?

Reply:
This is the code that takes a daffodil version and creaes the ivy config, which is used for the file name:

https://github.com/apache/daffodil-sbt/pull/2/files#diff-6ae6aa7ef1bba151de456d2a390adc7c0e0167448c9ac5401c1bb62018024a09R47-R49

That just removes all non `a-zA-Z0-9` characters from the version, so it should be `370SNAPSHOT`.